src/helpers/overrides.js

```diff
@@ -14,9 +14,14 @@ export type StyleOverrideObjectT = {};
 
 export type StyleOverrideT = StyleOverrideObjectT | StyleOverrideFunctionT;
 
+export type PropsOverrideFunctionT = ({}) => ?{};
+export type PropsOverrideObjectT = {};
+
+export type PropsOverrideT = PropsOverrideObjectT | PropsOverrideFunctionT;
```

Comment:
This type (and its sub-types) exactly duplicate `StyleOverrideT`. Can it be de-duplicated into a `ConfigurationOverrideT` and alias `export type StyleOverrideT = ConfigurationOverrideT`?